server/area_manager.py

```diff
@@ -647,7 +647,7 @@ def send_arup_status(self, clients=None):
             return
         status_list = [1]
         if len(self.server.hub_manager.hubs) > 1:
-            status_list = [1, "GAMING"]
+            status_list = [1, "HUB"]
```

Comment:
Pre-KFO Client might not display the right status color

Reply:
What version KFO client will be? so I can diversify from the old one?

Reply:
any client that isn't KFO LOL